Monika After Story/game/definitions.rpy

```diff
@@ -5500,6 +5500,18 @@ init -1 python:
 
         RETURNS: True if a proccess in proc_list is running, False otherwise
         """
+        if renpy.linux:
+            for proc in proc_list:
+                try:
+                    output = subprocess.call(["pidof", proc])
+
+                    # 0: Success
+                    # 1: Did not find
+                    if output == 0:
+                        return True
+                except:
```

Comment:
We should explicitly catch an exception type otherwise we might get stuff like `KeyboardInterrupt`.

So `except Exception:`

Also unsure if we really want to keep making subprocess calls when we failed one (esp since we don't log it). So maybe just `return False`